cnf-certification-test/platform/cnffsdiff/fsdiff.go

```diff
@@ -20,26 +20,27 @@ import (
 	"encoding/json"
 	"fmt"
 
+	mapset "github.com/deckarep/golang-set/v2"
 	"github.com/sirupsen/logrus"
 	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
 	"github.com/test-network-function/cnf-certification-test/pkg/testhelper"
 )
 
 var (
-	// targetFolders stores all the targetFolders that shouldn't have been modified in the container.
-	// All of them exist on UBI. It's a map just for convenience.
-	targetFolders = map[string]bool{
-		"/var/lib/rpm":  true,
-		"/var/lib/dpkg": true,
-		"/bin":          true,
-		"/sbin":         true,
-		"/lib":          true,
-		"/lib64":        true,
-		"/usr/bin":      true,
-		"/usr/sbin":     true,
-		"/usr/lib":      true,
-		"/usr/lib64":    true,
-	}
+	// targetFolders stores all the targetFolders that shouldn't have been
+	// modified in the container. All of them exist on UBI.
+	targetFolders = mapset.NewSet(
+		"/bin",
+		"/lib",
+		"/lib64",
+		"/sbin",
+		"/usr/bin",
+		"/usr/lib",
+		"/usr/lib64",
+		"/usr/sbin",
+		"/var/lib/rpm",
+		"/var/lib/dpkg",
+	)
```

Comment:
```suggestion
	targetFolders = []string{
		"/bin",
		"/lib",
		"/lib64",
		"/sbin",
		"/usr/bin",
		"/usr/lib",
		"/usr/lib64",
		"/usr/sbin",
		"/var/lib/rpm",
		"/var/lib/dpkg",
	}
```

Just switch this to a `[]string` and then we can use `StringInSlice`.

Reply:
Marking as resolved.